src/app/Library/CrudPanel/CrudField.php

```diff
@@ -150,6 +150,25 @@ public function size($numberOfColumns)
         return $this->save();
     }
 
+    /**
+     * Update the global CrudPanel object with the current field attributes.
+     * NOTE: Needs to be public so we can call it from FieldGroup.
+     *
+     * @return CrudField
+     */
+    public function save()
```

Comment:
sorry @pxpm for much comments( but i think no need touching CrudField.php now

Reply:
Np, you are right, just fixed.

Going to refactor for group :)